packages/portal-proto/src/utils/index.ts

```diff
@@ -284,3 +284,5 @@ export const statusBooleansToDataStatus = (
 
 export const focusStyles =
   "focus-visible:outline-none focus-visible:ring-offset-2 focus:ring-offset-white rounded-md focus-visible:ring-inset focus-visible:ring-2 focus-visible:ring-focusColor";
+
+export const LG_BREAKPOINT = 1024;
```

Comment:
Can we import this from the tailwind theme? I think its a good idea to just have one source of truth for what the breakpoint values are.

`export const LG_BREAKPOINT = parseInt(tailwindConfig.theme.extend.screens.lg)`

Reply:
I agree. Thank you for mentioning.